packages/@aws-cdk/aws-appsync/lib/graphqlapi-base.ts

```diff
@@ -0,0 +1,146 @@
+import { ITable } from '@aws-cdk/aws-dynamodb';
+import { IFunction } from '@aws-cdk/aws-lambda';
+import { CfnResource, IResource, Resource } from '@aws-cdk/core';
+import { DynamoDbDataSource, HttpDataSource, LambdaDataSource, NoneDataSource } from './data-source';
+
+/**
+ * Interface for GraphQL
+ */
+export interface IGraphQLApi extends IResource {
+
+  /**
+   * an unique AWS AppSync GraphQL API identifier
+   * i.e. 'lxz775lwdrgcndgz3nurvac7oa'
```

Comment:
nice! can you add proliferate this everywhere in this PR where you have a doc string for API id?